### PR TITLE
feat: added initial implementation

### PR DESCRIPTION
**Pull request by Rasakul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,82 +23,115 @@
...
-    // Configure the public key to authenticate with the API endpoint
-    options.publicKey = "<YOUR PUBLIC APP KEY>"
+### Apps built with SwiftUI
 
-    // (Optional) Configure a custom base URL to your API host
-    // options.baseURL = "https://your-domain.com/api"
-}
-...
-```
+TODO
```
Comment (philprime) [inline README.md:61]: Please add a description how to integrate OnLaunch

**Review by philprime — CHANGES_REQUESTED** [review]
Can we remove the `coverage_badge.svg`? I do not believe it should be checked in.

**Comment by Rasakul** [comment]
@philprime true, thanks for noticing! Fixed

**Comment by philprime** [comment]
Instead of removing the description I would have added in the unit tests again, but if they are unnecessary, removing them is fine too.

I removed the whole section about "Continuous Integration" from the README you missed to remove.

**Review by philprime — APPROVED** [review]

**Comment by Rasakul** [comment]
@philprime there are no unit tests for the current version. I'd add the badge later again, when integrating the unit tests.

Thanks for fixing the Readme (again 😅)

**Pull request merged**